editor/project_settings_editor.cpp

```diff
@@ -233,7 +233,7 @@ void ProjectSettingsEditor::_device_input_add() {
 			Ref<InputEventJoypadMotion> jm;
 			jm.instance();
 			jm->set_axis(device_index->get_selected() >> 1);
-			jm->set_axis_value(device_index->get_selected() & 1 ? 1 : -1);
+			jm->set_axis_value(device_index->get_selected() & 1 ? device_special_value->get_value() : -device_special_value->get_value());
```

Comment:
You ignored my recommendation of using 1-strength here. This code will break older projects, which will suddendly have a deadzone of 1. Please use 1-strength.